crawl-ref/source/dat/des/branches/abyss.des

```diff
@@ -1637,6 +1637,52 @@ ccBBBccc-cRc-cccDDDcc
 ccccccccccccccccccccc
 ENDMAP
 
+NAME:   spicy_pingpong_abyss_rune
```

Comment:
I think the monster density on this is too high. The player is going to be eating a lot of starflower hits to try for this rune, each of which has a good chance of sending them deeper (away from the rune vault) or if they find it on A:5 resetting things.